core/agents/orchestrator.py

```diff
@@ -163,11 +164,23 @@ def create_agent(self, prev_response: Optional[AgentResponse]) -> BaseAgent:
             if prev_response.type == ResponseType.INPUT_REQUIRED:
                 # FIXME: HumanInput should be on the whole time and intercept chat/interrupt
                 return HumanInput(self.state_manager, self.ui, prev_response=prev_response)
-            if prev_response.type == ResponseType.UPDATE_EPIC:
-                return TechLead(self.state_manager, self.ui, prev_response=prev_response)
             if prev_response.type == ResponseType.TASK_REVIEW_FEEDBACK:
                 return Developer(self.state_manager, self.ui, prev_response=prev_response)
 
+        status = state.current_task.get("status") if state.current_task else None
+        if status:
+            # Status of the current task is set first time after the task was reviewed by user
+            log.info(f"Status of current task: {status}")
+            if status == "reviewed":
```

Comment:
Can we have all the statuses listed TaskStatus.REVIEWED ... (etc), (same like we did with specification.Complexity?) Helps with us poor typo-friendly devs :D  No need to subclass Enum (since it complicates things a bit, hence I didn't do that when converting Complexity), but just having those as constants will make me sleep easier at night.

Reply:
haha anything for you, will change it